libcextract/NonLLVMMisc.cpp

```diff
@@ -140,3 +140,15 @@ enum FileHandling::FileType FileHandling::Get_File_Type(int fd)
 
   return FILE_TYPE_UNKNOWN;
 }
+
+/** Get basename of a string.  Works like the gnu version.  */
+const char *get_basename(const char *filename)
+{
+#if defined(_WIN32) || defined(WIN32)
+  const char delim = '\\';
```

Comment:
`/` is a valid path delimiter on WIN32 - both need to be checked. E.g. here's our (JuliaLang's) version: https://github.com/JuliaLang/julia/blob/fa66b63fc3421fa549f901afeb88d34dc88d06fd/src/timing.h#L8-L17

Reply:
I will just copy this one then.